src/components/Homes/Home.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import NavBar from '../Nav/Navbar';
+import Article from '../Articles/Article';
+import Tags from '../Tag/Tags';
+import Footer from '../Footers/Footer';
```

Comment:
Foote is declared in this file but never used. Same goes for NavBar. Please remove them. This is also an indication that either your Eslint is not properly configured or you chose to ignore the eslint warnings.

Reply:
I have reacted to this, please.